api/v1alpha4/types.go

```diff
@@ -700,6 +700,10 @@ type Volume struct {
 	// +optional
 	IOPS int64 `json:"iops,omitempty"`
 
+	// Throughput to provision in MiB/s supported for the volume type. Not applicable to all types.
```

Comment:
Do you think its worth mentioning in the comment the types its applicable to or providing a link to the AWS docs?

Reply:
Alright, I added a link next to the documentation for VolumeTypes. I can add it here as well if you prefer.